### DISCUSSION TITLE
Any documentation/explanation for the new settings?

### DISCUSSION THREAD

**Discussion by boredsebastian** [hako-mikan/sd-webui-supermerger#194] [Q&A] [answered]
![image](https://github.com/hako-mikan/sd-webui-supermerger/assets/45526702/4fa8d621-ea7c-410f-845b-bb4ea88b69b1)
What does all that even mean?
And is the merging process the same as before if I leave those settings at default values?

**Comment by miasik** [comment, marked as answer]
Mergings are the same. These new controls help to set different block to one value with one click.

**Reply by boredsebastian** [reply]
Oh I see now.
I actually wanted the option to set everything to a value, add, subtract, divide and multiply long ago so I made a plain HTML - JS page only for that lol, can also make linear gradients
![image](https://github.com/hako-mikan/sd-webui-supermerger/assets/45526702/0122e451-536b-4a1e-8dfb-5dfcaf7ba967)
And actually moving the sliders and all calculations set the weights directly into the textbox, so you save some clicks and minimize errors by forgetting to manually pressing "set to alpha"

**Reply by miasik** [reply]
well, I'd like to have something like formulas for dynamic calculation.
IN00=OUT00/2
When we change OUT00, IN00 gets changed according to the formula.
